apps/browser/qml/pages/PermissionExceptionsPage.qml

```diff
@@ -92,7 +96,24 @@ Page {
                 MenuItem {
                     //% "Delete"
                     text: qsTrId("sailfish_browser-me-delete")
-                    onClicked: remove(model.index)
+                    onClicked: remove(model.uri, model.type, model.capability)
+                }
+            }
+        }
+
+        PullDownMenu {
+            visible: model && (view.count > 0)
+            MenuItem {
+                //% "Delete all exceptions"
+                text: qsTrId("sailfish_browser-me-delete-all-exceptions")
```

Comment:
visible: model && (view.count > 0)

After moving to the other pull down menu... add visible binding to the menu item itself.